sdk/rust/src/consensus/engine.rs

```diff
@@ -32,7 +32,7 @@ pub enum Update {
     BlockCommit(BlockId),
 }
 
-#[derive(Default, Debug)]
+#[derive(Clone, Default, Debug)]
```

Comment:
This should probably be a separate commit.

Reply:
Agreed